pkg/odo/cli/component/delete.go

```diff
@@ -28,10 +28,24 @@ import (
 // DeleteRecommendedCommandName is the recommended delete command name
 const DeleteRecommendedCommandName = "delete"
 
-var deleteExample = ktemplates.Examples(`  # Delete component named 'frontend'. 
+var deleteExample = ktemplates.Examples(`
+# Delete the component present in the current directory
+%[1]s
+
+# Delete the component named 'frontend'
 %[1]s frontend
-%[1]s frontend --all
-  `)
+
+# Delete the component and all the related config files from the current directory
```

Comment:
```suggestion
# Delete the component and all the related config files ("devfile.yaml" and ".odo" directory) from the current directory
```
Reason for this suggestion is that the user might misunderstand it as some other configuration in their pwd